app/graphql/types/Input/route_input_type.rb

```diff
@@ -0,0 +1,12 @@
+module Types
+  class Input::RouteInputType < Input::BaseInput
+    description "Attributes for creating or updating a route"
+    argument :userEmail, Integer, required: false
```

Comment:
Should userEmail be an Integer? Elsewhere it's a string and that seems to make sense.

Reply:
Excellent catch.  This was not being used yet (no authorization implemented), that why the test didn't catch it.

Reply:
Do we need to fix it before merging this PR or do you want to add it to a bug list?